daccs_client/services.py

```diff
@@ -16,8 +16,6 @@ def __init__(self, servicejson: dict[str, Any]) -> None:
 
         self._service = None
         self._service_doc = None
-        self._service_desc = None
```

Comment:
As discussed previously, these are optional so probably shouldn't be hardcoded as options